bumble/pandora/host.py

```diff
@@ -735,6 +738,18 @@ def uuid16_from_str(uuid: str) -> bytes:
                 )
             )
 
+        flag_map = {
+            NOT_DISCOVERABLE: AdvertisingData.BR_EDR_NOT_SUPPORTED_FLAG,
+            DISCOVERABLE_LIMITED: AdvertisingData.LE_LIMITED_DISCOVERABLE_MODE_FLAG,
+            DISCOVERABLE_GENERAL: AdvertisingData.LE_GENERAL_DISCOVERABLE_MODE_FLAG,
+        }
+
+        if dt.le_discoverability_mode:
```

Comment:
This one is a bit special, should we always set it?

I don't remember how protobuf works but I think here you are exclusing the `NOT_DISCOVERABLE` value

Reply:
Oh, per the spec it seems that we can exclude the structure when all flags are 0

```
The Flags data type shall be included when any of the Flag bits are non-zero and the advertising packet is connectable, otherwise the Flags data type may be omitted.
```